interface/orders/types.php

```diff
@@ -192,6 +191,9 @@ function refreshFamily(id, haskids) {
     event.preventDefault();
     console.log(event.target.href);
     $("#modal-iframe").iziModal({
+    title: 'Add procedure',
+    subtitle: 'Fill in procedure',
```

Comment:
Translations?  Meaning use of xlt() function.

Reply:
yes use those i forgot thanks @aethelwulffe

Reply:
Sorry but I do not understand fully.

Reply:
xlt() is a translation function @mshubhankar so basically just pass the texts as parameter to it so it displays a correct message in the appropriate language